backend/queries/fellows.js

```diff
@@ -50,37 +51,54 @@ const getUserByEmail = async (fEmail) => {  // function name breaks convention f
   return await db.one(getQuery, { fEmail });
 }
 
-// const addFellow = async (bodyObj) => {
-//   const postQuery = `
-//     INSERT INTO fellows (
-//         f_first_name,
-//         f_last_name,
-//         f_picture,
-//         f_bio,
-//         f_linkedin,
-//         f_github,
-//         cohort_id,
-//         want_mentor
-//     ) VALUES (
-//         $/firstname/,
-//         $/lastname/,
-//         $/picture/,
-//         $/bio/,
-//         $/linkedIn/,
-//         $/github/,
-//         $/cohortId/,
-//         $/wantMentor/
-//     )
-//     RETURNING *;
-//   `;
-//   return await db.one(postQuery, bodyObj);
-// }
+const addFellow = async (user, password) => {
+  await userQueries.addUser(user.email, password, 'fellow');
+
+  const postQuery = `
+    INSERT INTO fellows (
+        f_first_name,
+        f_last_name,
+        f_email,
+        cohort_id
+    ) VALUES (
+        $/firstName/,
+        $/lastName/,
+        $/email/,
+        $/cohortId/
+    )
+    RETURNING *;
+  `;
+  return await db.one(postQuery, user);
+}
+
+const updateFellow = async (user) => {
+  const updateQuery = `
+    UPDATE fellows
+    SET 
+      f_first_name = $/firstName/,
+      f_last_name = $/lastName/,
+      f_picture = $/picture/,
+      f_bio = $/bio/,
+      f_linkedin = $/linkedIn/,
+      f_github = $/github/,
+      cohort_id = $/cohortId/,
+      want_mentor = $/wantMentor/
+    WHERE f_id = $/userId/
+    RETURNING *;
+  `;
+}
+
+const deleteFellow = async (id) => {
+  return await db.one('DELETE FROM fellows WHERE v_id = $/id/ RETURNING *', {id});
```

Comment:
this should be WHERE f_id, not v_id. I'll copy over, fix this typo, and finish hashing out the add, update, and deletes in my next pull req so just be on the lookout for the duplicate looking code